pkg/daemon/daemon.go

```diff
@@ -146,13 +150,19 @@ const (
 	// the MCC/MCO will time out.  We don't want to spam our logs with the same
 	// error.
 	updateDelay = 5 * time.Second
+
 	// maxUpdateBackoff is the maximum time to react to a change as we back off
 	// in the face of errors.
 	maxUpdateBackoff = 60 * time.Second
+
+	// How frequently to check for config drift
+	configDriftCheckInterval = 60 * time.Minute
```

Comment:
@sinnykumari are you ok with this timing? I thought you wanted longer?

Reply:
not really, this needs discussion within team so that we all agree on a particular interval. Let's do this in one of the meeting.